jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/composite/page/init/PageInitTests.java

```diff
@@ -13,10 +13,10 @@
 import static page.objects.selenium.PageBase.JDI_DOMAIN;
 
 public class PageInitTests {
+    private boolean first = true;
     private static String expecterdUrl(String url) {
```

Comment:
Please, add empty lines around this method

Reply:
done:

public class PageInitTests {

    private boolean first = true;

    private static String expecterdUrl(String url) {
        return JDI_DOMAIN + url;
    }
    
    @BeforeMethod
    public void before() {